docs/features/DefaultInterfaceImplementation.md

```diff
@@ -51,6 +51,8 @@ class Test1 : I1
 
 - Declaring static fields, auto-properties and field-like events (**protected** modifier is not allowed).
 
+- Declaring operators ```+ - ! ~ ++ -- true false * / % & | ^ << >> > < >= <=``` in interfaces.
```

Comment:
> Is the intention to support equality later on? If not, maybe add a note as to why it is not allowed.

This is a topic for LDM discussion. The list is meant to simply enumerate what is supported by the prototype without any reasoning, and it doesn't have to match the proposed spec.